tests/drivers/test_forecast.py

```diff
@@ -165,3 +166,29 @@ def test_create_directory_structure():
             forecast_obj.create_directory_structure(run_directory, "quit")
         assert pytest_wrapped_e.type == SystemExit
         assert pytest_wrapped_e.value.code == 1
+
+def test_create_forecast_obj():
+    ''' Tests that stage_fixed_files() is copying files from config_obj 
+    are being staged in run directory.'''
+
+    with tempfile.TemporaryDirectory() as run_directory:
+
+        test_yaml = os.path.join(uwtools_file_base,pathlib.Path("../fixtures/expt_dir.yaml"))
+        test_cfg = config.YAMLConfig(test_yaml)
+        forecast_obj = FV3Forecast()
+
+        forecast_obj.create_directory_structure(run_directory)
+
+        files_to_stage = test_cfg.get('static', {})
+
+        for target, file_path in files_to_stage.items():
+            pathlib.Path(file_path).touch()
+
+        forecast_obj.stage_fix_files(run_directory, test_cfg)
+
+        for target, file_path in files_to_stage.items():
+            assert os.path.isfile(file_path)
+            file = os.path.join(run_directory, target)
+            assert os.path.isfile(file)
+            assert filecmp.cmp(file_path, file)
+            os.remove(file_path)
```

Comment:
I'd suggest adopting `src` / `dst` and `fn` / `path` terminology here, too. Differentiating between filenames, paths, and actual file objects (like what `open()` returns) is a perennial issue in Python codes. Adopting a convention to make it clear which kind of thing you're dealing with is helpful. (Type annotations help distinguish file objects from the other two, but filenames and paths are both `str` objects, so it doesn't help there.)